datafusion/physical-plan/src/aggregates/mod.rs

```diff
@@ -2032,7 +2037,7 @@ mod tests {
         spill: bool,
     ) -> Result<()> {
         let task_ctx = if spill {
-            new_spill_ctx(2, 3200)
+            new_spill_ctx(2, 4200)
```

Comment:
Again, seems like we need to make something configurable here?

Reply:
I might be misunderstanding, do you mean the change in 3a33755527a0a602e32fe6cfc1e1476a5949a48d 🤔 ?